### ISSUE TITLE
Analysis: Time spent auditing (Audit Time) vs Labeling frequency

### ISSUE THREAD

**Issue by manaswis** [ProjectSidewalk/sidewalk-data-analysis#33]
We should add this relationship analysis in the behavioral analysis section as well since we talk about it later in the user group comparison section.

While you are at it, also check for _Audit time vs Accuracy_ as well.

So to summarize, we will now four attributes: visual search time, labeling frequency, auditing speed and visual search time in the user behavioral section.

**Comment by misaugstad** [comment]
I think there are a couple typos with the issue as stated, so I'm not quite sure what you are looking for.

By audit time, do you mean auditing speed..? And you say there are 4 attributes, but you list one of them twice in your list of 4 :)

**Comment by manaswis** [comment]
> By audit time, do you mean auditing speed..? 

No, it is the time spent auditing in minutes. Refer to the descriptive stats for user table [here](https://github.com/ProjectSidewalk/sidewalk-data-analysis/blob/master/R/stats_for_paper.md#descriptive-stats-for-users). The columns are "mins.audited" and "mins.audited.std". 
"Audit Time" is the term you came up with when we were making the table 4 in the paper compact :)

> By audit time, do you mean auditing speed..? And you say there are 4 attributes, but you list one of them twice in your list of 4 :)

Oops! I meant audit time at the end. Updated the issue post.

**Comment by misaugstad** [comment]
Okay we shouldn't be using total time spent auditing as anything other than a descriptive stat. Different users audited different distances, so the straight up total time auditing is not a good measure. That is why I used auditing speed (time / distance).
